data/localized/de_patch/text/conversations/10_od_nua/1015_cv_adra_dragon.stringtable

```diff
@@ -383,7 +383,7 @@ Der Drache harkt mit seinen Klauen über die Erde. „Natürlich liegen hier meh
     </Entry>
     <Entry>
       <ID>113</ID>
-      <DefaultText>„Gewissermaßen ja. Stell dir nur vor&#160;– wie viele hundert Seelen, getötet von kriecherischen Xaurips oder von Trollen verschlungen wurden sind. Gegen versiegelte Türen geschmettert, schreiend von Spinnen in dunkle Gruben gezogen. Alle, um zu sehen, was du gesehen hast.“</DefaultText>
+      <DefaultText>„Gewissermaßen ja. Stell dir nur vor&#160;– wie viele hundert Seelen, getötet von kriecherischen Xaurips oder von Trollen verschlungen worden sind. Gegen versiegelte Türen geschmettert, schreiend von Spinnen in dunkle Gruben gezogen. Alle, um zu sehen, was du gesehen hast.“</DefaultText>
```

Comment:
How many hundreds of souls, killed by fawning xaurips or devoured by trolls.

Vermutlich abwertend gemeint.